tests/functionNameTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require_once 'src/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class functionNameTest extends TestCase{
+    public function testDogInfoIsCorrect(): void
+    {
+
+        $dogs = [[
+            'name' => 'Rolly',
+            'breed' => 'King Charles',
+            'owner_id' => 1,
+            'image' => 'rolly.jpeg'
+        ]];
+
+        $actual = returnDogs($dogs);
+
+        $expected = '<div class="collection_item">' .
+            '<h4>Dog name:</h4>' . '<p>Rolly</p>' .
+            '<h4>Dog breed:</h4>' . '<p>King Charles</p>' .
+            '<em>I belong to Nan</em>' .
+            '</div>  
+            <img class="collection_item_image" src="rolly.jpeg">';
+
+        $this->assertEquals($expected, $actual);
+    }
```

Comment:
This test isn't testing that the info is correct, it is testing that your displayer function puts out the right HTML - again naming here could be better.

What if your dog array has some of the fields, but not all of them? Are all fields required?